timemachine/potentials/generic.py

```diff
@@ -439,6 +439,12 @@ def from_gpu(p: gpu.CustomOpWrapper) -> Optional[Potential]:
     if isinstance(p, gpu.Nonbonded):
         return Nonbonded.from_gpu(p)
 
+    if isinstance(p, gpu.NonbondedAllPairs):
```

Comment:
In some exploration I noticed that these two were missing from the generic and added them. Can easily remove these, just left them as they seemed like nice to haves.

Reply:
LGTM (I'd just been adding things to `generic` as I found them useful)